packages/ui/src/components/Markdown/index.ts

```diff
@@ -0,0 +1,7 @@
+import { Markdown } from './markdown'
+import './style.less'
+
+export type { MarkdownProps } from './markdown'
```

Comment:
I think you should remove `level` prop from this type.
For example:
```suggestion
Omit<MarkdownProps, 'level'>
```